spec/integration/test_runner_spec.rb

```diff
@@ -78,7 +78,7 @@ def verbosely_log(object, label)
     it 'process all worksheets and does not raise any error' do
       results = []
       worksheet_names.each do |spreadsheet_name|
-        # next unless spreadsheet_name == 'Test - P3 - Test1'
+        puts ">>>>>>>>> #{spreadsheet_name} #{__FILE__}:#{__LINE__} <<<<<<<<<<\n"
```

Comment:
Debug statement left in...

Reply:
Also, does spec/fixtures/integration_test_data.xlsx **need** to be an xlsx file?  
It needs specific software to edit/update it, could it be replaced with a CSV?

Reply:
These are now controlled by VERBOSE env var, set either to true for mildly verbose output, or noisy for very verbose output.  Nothing is printed if env var not set.

Yes, it does need to be ab .xlsx file because there is one tab for each test scenario, but it is created with Google sheets.